src/Utilities/GameConfiguration.cs

```diff
@@ -46,7 +46,7 @@ private static void Initialize()
             Copyright = assembly.GetCustomAttribute<AssemblyCopyrightAttribute>().Copyright.Replace("©", "(c)");
 
             // Additional work to modify raw settings should generally be done once and cached, as follows.
-            string root = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData);
+            string root = System.Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
```

Comment:
It's probably copy pasta on my part. I'm using Visual Studio for Mac 8.9 beta and everything worked surprisingly well. They've really made great strides improving the IDE experience lately. I'm more dev-ops these days than dev (Can I interest you in yaml files?) I never have the pleasure of working at a long stretch in any one language or even platform these days so that means I'm marginally competent at whatever I end up having to do in the moment. That's probably what's to blame for my eyeballs not being properly trained to spot this sort of error and not having proper tooling in place to bring it to my attention.